src/pytraits/core/composing/compiler.py

```diff
@@ -39,7 +40,8 @@ def _clone_function(self, function):
 
         trait["co_argcount"] = function.__code__.co_argcount
         trait["co_kwonlyargcount"] = function.__code__.co_kwonlyargcount
-        trait["co_posonlyargcount"] = function.__code__.co_posonlyargcount
+        if sys.version_info[0] == 3 and sys.version_info[1] >= 8:
```

Comment:
`sys.version_info[:2] >= (3, 8)` is more common.

Reply:
Indeed, this is a nice and compact test.